File: chat-uikit/src/main/java/io/agora/chat/uikit/chat/model/EaseReplyMap.java
package io.agora.chat.uikit.chat.model;

/**
 * Map the message type to {@link io.agora.chat.ChatMessage.Type}
 */
public enum EaseReplyMap {
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#TXT}
     */
    txt,
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#IMAGE}
     */
    img,
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#VIDEO}
     */
    video,
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#LOCATION}
     */
    location,
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#VOICE}
     */
    audio,
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#FILE}
     */
    file,
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#CMD}
     */
    cmd,
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#CUSTOM}
     */
    custom,
    /**
     * Map to {@link io.agora.chat.ChatMessage.Type#COMBINE}
     */
    combine,
    unknown
}

File: ease-im-kit/src/main/kotlin/io/agora/uikit/common/enums/EaseReplyMap.kt
package io.agora.uikit.common.enums

/**
 * Map the message type to [io.agora.chat.ChatMessage.Type]
 */
enum class EaseReplyMap {
    /**
     * Map to [io.agora.chat.ChatMessage.Type.TXT]
     */
    txt,

    /**
     * Map to [io.agora.chat.ChatMessage.Type.IMAGE]
     */
    img,

    /**
     * Map to [io.agora.chat.ChatMessage.Type.VIDEO]
     */
    video,

    /**
     * Map to [io.agora.chat.ChatMessage.Type.LOCATION]
     */
    location,

    /**
     * Map to [io.agora.chat.ChatMessage.Type.VOICE]
     */
    audio,

    /**
     * Map to [io.agora.chat.ChatMessage.Type.FILE]
     */
    file,

    /**
     * Map to [io.agora.chat.ChatMessage.Type.CMD]
     */
    cmd,

    /**
     * Map to [io.agora.chat.ChatMessage.Type.CUSTOM]
     */
    custom,

    /**
     * Map to [io.agora.chat.ChatMessage.Type.COMBINE]
     */
    combine,
    unknown
}